cmake/UraniumTranslationTools.cmake

```diff
@@ -108,6 +108,6 @@ if(GETTEXT_FOUND)
         TARGETS_FOR_PO_FILES(${language})
         TARGETS_FOR_MO_FILES(${language})
     endforeach()
-    install(DIRECTORY ${CMAKE_BINARY_DIR}/resources DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium/)
+    install(DIRECTORY ${CMAKE_BINARY_DIR}/resources DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/)
```

Comment:
No.

1. I'm already using it above and it works fine
2. As far as I understand https://cmake.org/Wiki/CMake_Useful_Variables "CMAKE_PROJECT_NAME" would be "cura-build" if we build Uranium inside cura-build, as cura-build would be the "the top level project". Isn't it?